app/controllers/application_controller.rb

```diff
@@ -1,2 +1,7 @@
 class ApplicationController < ActionController::Base
+    
+    
```

Comment:
Delete one of the two spaces you left.

Reply:
Thanks i will be more careful with the indentation and spaces issues. I already made this change.